get_github_coderabbit_review/auth/rbac.py

```
@@ -0,0 +1,996 @@
+    Returns:
+        Decorator function
+    """
+
+    def decorator(func):
+        @functools.wraps(func)
+        def wrapper(*args, **kwargs):
+            # Get user_id from context (implement based on your auth system)
+            user_id = kwargs.get("user_id") or getattr(args[0], "user_id", None)
+
+            if not user_id:
+                raise PermissionError("User not authenticated")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)